bin/generateChromeManifest.py

```diff
@@ -65,11 +61,7 @@ def main(l10n_src, input_file_name, output_file_name):
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(
-        description="Loop Stand-alone Client localization update script")
-    parser.add_argument("--src",
-                        default=DEF_L10N_SRC,
-                        metavar="path",
-                        help="Source path for l10n resources. Default = " + DEF_L10N_SRC)
+        description="Loop chrome manifest generation script")
```

Comment:
That's `Loop Stand-alone Client localization update script` which I think is fine already.